### PR TITLE
fix(onboarding): Add checking passwords to new password input

### PR DESCRIPTION
**Pull request by borismelnik**
Closes: #5834

### What does the PR do

Add passwords validation to new password input

### Affected areas

Onboarding

### CODE REVIEW

File: ui/imports/shared/views/PasswordView.qml [hunk 1]
```diff
@@ -222,6 +222,10 @@ Column {
 
                 // Update strength indicator:
                 strengthInditactor.strength = d.convertStrength(RootStore.getPasswordStrengthScore(newPswInput.text, root.onboarding))
+
+                if(textField.text.length === confirmPswInput.text.length || textField.text.length === 0) {
```
Comment (PascalPrecht) [inline ui/imports/shared/views/PasswordView.qml:226]: Do we really want to check whether the password matches when the `textField.text.length === 0`?
At that point we already know there's practically no password at all

Comment (stefandunca) [reply ui/imports/shared/views/PasswordView.qml:226]: I'm not sure I understand the purpose for this check `if(textField.text.length === confirmPswInput.text.length || textField.text.length === 0)`.
It looks to me that both checks are done in `checkPasswordMatches`, and this "optimization" doesn't make any improvement or do I miss something?

**Review by stefandunca — APPROVED** [review]

File: ui/imports/shared/views/PasswordView.qml (hunk 1 above)
Comment (borismelnik) [reply ui/imports/shared/views/PasswordView.qml:226]: Guys I don't know actually, I just moved some checks from other component. But I will check and fix if it's possible.

Comment (borismelnik) [reply ui/imports/shared/views/PasswordView.qml]: I fixed it

**Review by iurimatias — APPROVED** [review]

**Pull request merged**